docs/utils/select-atom.mdx

```diff
@@ -53,4 +53,6 @@ const nameAtom = selectAtom(personAtom, (person) => person.name)
 const birthAtom = selectAtom(personAtom, (person) => person.birth, deepEquals)
 ```
 
-Ref: https://codesandbox.io/s/react-typescript-forked-8czek
+## CodeSandbox
+
+<CodeSandbox id="8czek" />
```

Comment:
Enjoyed your examples, we should keep them! but again, please also remove unused code! like unused imports! 
We appreciate your help!